File: 1.two sum.go
package twosum

func twoSum(nums []int, target int) []int {
	for i := 0; i < len(nums); i++ {
		for l := i + 1; l < len(nums); l++ {
			if nums[i]+nums[l] == target {
				return []int{i, l}
			}
		}
	}
	return nil
}

File: 1.two sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        nums_map = {}

        for i, num in enumerate(nums):
            diff = target - num
            if diff in nums_map:
                return [nums_map[diff], i]
            nums_map[num] = i

        return []